src/wireguard/handshake/messages.rs

```diff
@@ -239,7 +240,8 @@ impl fmt::Debug for NoiseInitiation {
 #[cfg(test)]
 impl fmt::Debug for NoiseResponse {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f,
+        write!(
+            f,
             "NoiseResponse {{ type = {}, sender = {}, receiver = {}, ephemeral = {}, empty = |{}  }}",
             self.f_type,
             self.f_sender,
```

Comment:
I actually think the cleaner thing might be: https://doc.rust-lang.org/std/fmt/struct.DebugStruct.html